agent/agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -473,7 +473,7 @@ func (m *PGStatMonitorQAN) makeBuckets(current, cache map[time.Time]map[string]*
 
 			if !m.disableQueryExamples && currentPSM.Example != "" {
 				mb.Common.Example = currentPSM.Example
-				mb.Common.ExampleFormat = agentpb.ExampleFormat_EXAMPLE
+				mb.Common.ExampleFormat = agentpb.ExampleFormat_EXAMPLE // nolint:staticcheck
```

Comment:
Can you please create ticket for that. Otherwise we will forget about that problem.